apps/web/src/components/Icon/Icon.tsx

```diff
@@ -0,0 +1,25 @@
+import * as React from 'react';
+import './Icon.scss';
+
+export enum iconColor {
+  Default = 'black',
+  Orange = '#ff2800',
+}
+
+interface Props {
+  children: React.ReactElement;
+  colorProp?: iconColor;
+}
+
+/**
+ * Renders an icon component.
+ *
+ * @param {Object} props - The component props.
+ * @param {React.ReactElement} props.children - The child element to render within the icon.
+ * @returns {JSX.Element} - A JSX element representing the icon component.
+ */
+const Icon = ({ children, colorProp }: Props) => {
+  return <div style={{ color: colorProp }}>{children}</div>;
```

Comment:
I think it's best practice to add a default value for optional props, especially when using Storybook. This way the `default` story only passes props that are required. 

```suggestion
  return <div style={{ color: colorProp = ColorsEnum['some-color']}}>{children}</div>;
```